components/MaskedTransition/tests/unit/MaskedTransitionNoopTest.m

```diff
@@ -26,8 +26,9 @@ @implementation MaskedTransitionNoopTest
 
 - (void)testInit {
   UIView *view = [[UIView alloc] init];
-  MDCMaskedTransition *transition = [[MDCMaskedTransition alloc] initWithSourceView:view];
-  XCTAssertNotNil(transition);
+  MDCMaskedTransitionController *transitionController =
+      [[MDCMaskedTransitionController alloc] initWithSourceView:view];
```

Comment:
Super-duper-nit: Test that `view` and `transitionController.sourceView` are the same object or else call the designated initializer `-init` without a sourceView.

Reply:
Done.